### ISSUE TITLE
Check out Volla Phone

### ISSUE THREAD

**Issue by probonopd** [helloSystem/hello#218]
Check out https://volla.online/en/index.html

* They are selling a smartphone made in Germany
* Focused on privacy (like helloSystem)
* Focused on simplicity (e.g., no notifications - like helloSystem)
* Factory-loaded with open source software
* Supporting the UBports project (which is using Qt - like helloSystem)

Their philioshopy seems to align well with ours, down to the similar logo and the name of their company, "Hallo Welt Systeme UG".

Article (in German):
https://rp-online.de/nrw/staedte/remscheid/remscheider-entwickelt-neues-smartphone_aid-49279221

It may be a good match (phone) for helloSystem (desktop).

**Comment by probonopd** [comment]
First challenge probably would be to get the UBports Installer running on helloSystem.

https://github.com/ubports/ubports-installer/issues/2211

**Comment by wurzer** [comment]
A kind of that is our plan. What do you think about convergence?

**Comment by wurzer** [comment]
I think the idea is to get fewer and only relevant messages. It takes me a lot of effort in macOS to disable or limit the notifications from the many apps. The issue could be extended to function tiles of apps. For example, it is not possible in macOS to synchronize e-mails with the default program only manually. However, this is necessary for concentrated and productive work.

**Comment by probonopd** [comment]
Exactly. The user should tell the computer what to do, not the other way around. Same for the phone. imho.
